### PR TITLE
Latest Changes

### PR DESCRIPTION
**Pull request by LuisBanderasB**

### CODE REVIEW

File: app/controllers/boards_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  end
+
+  private 
+
+  def board_params 
+    params.require(:board).permit(:name, :visibility)
+  end
+
+  def set_board
+    @board = Board.find(params[:id])
```
Comment (ohenroute) [inline app/controllers/boards_controller.rb]: what if this throws an error?

File: app/controllers/boards_controller.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+class BoardsController < ApplicationController
+  before_action :set_board, only: [:show]
+
+  def index
+    @boards = Board.all
+  end
+
+  def show
+    @lists = @board.lists
```
Comment (ohenroute) [inline app/controllers/boards_controller.rb]: why not adding this to the presenter?

**Review by ohenroute — CHANGES_REQUESTED** [review]

File: app/javascript/packs/application.js [hunk 3]
```diff
@@ -7,7 +7,11 @@ import Rails from "@rails/ujs"
...
 import * as ActiveStorage from "@rails/activestorage"
 import "channels"
+import "../stylesheets/application"
 
 Rails.start()
 Turbolinks.start()
 ActiveStorage.start()
+require("jquery")
+require("bootstrap")
+require("dragula")
```
Comment (ohenroute) [inline app/javascript/packs/application.js]: use all imports or all requires.

**Review by ohenroute — CHANGES_REQUESTED** [review]

File: app/presenters/board_presenter.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+class BoardPresenter
+  def initialize(board)
+    @board = board
+  end
+  def lists
+    @board.lists
+  end
+end
```
Comment (ohenroute) [inline app/presenters/board_presenter.rb:8]: Add space between methods.

File: app/views/boards/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+          <h4><%= list.name %></h4>
+          <ul class="list-group">
+            <li class="list-group-item">An item</li>
+            <li class="list-group-item">A second item</li>
+            <li class="list-group-item">A third item</li>
+            <li class="list-group-item">A fourth item</li>
+            <li class="list-group-item">And a fifth one</li>
+          </ul>
+        </div>
+      <% end %>
```
Comment (ohenroute) [inline app/views/boards/show.html.erb:17]: Don't you think this could go in a partial? Abstract it.

**Review by ohenroute — APPROVED** [review]
Approved, but please make the additional changes.

File: app/views/boards/show.html.erb (hunk 5 above)
Comment (LuisBanderasB) [reply app/views/boards/show.html.erb:17]: I've already done that on the current branch I'm working

**Pull request merged**